<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>check_table</class>
 <widget class="QWidget" name="check_table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>TABLES</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>01</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>02</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>04</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_20">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>03</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>05</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>06</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>07</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>08</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>09</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>260</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>19</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_19">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>340</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>information</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>90</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;预约状态：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>120</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;预约剩余时间：&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;已预约&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>120</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>101</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;桌号：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>40</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;01&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>170</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;使用状态：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>170</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;已落座&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_22">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>230</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
         </property>
         <property name="text">
          <string>预约</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_23">
         <property name="geometry">
          <rect>
           <x>174</x>
           <y>230</y>
           <width>91</width>
           <height>41</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
         </property>
         <property name="text">
          <string>落座</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;请再次点击&lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#ff007f;&quot;&gt;桌子按钮&lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;或&lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#00aa7f;&quot;&gt;刷新键&lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;以刷新桌子信息：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#0055ff;&quot;&gt;桌子颜色，预约剩余时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_21">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
           </property>
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_24">
           <property name="font">
            <font>
             <family>微软雅黑</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> QPushButton{
                background-color:rgba(100,225,100,30);//背景色（也可以设置图片）
  border-style:outset; //边框样式（inset/outset）
  border-width:4px;  //边框宽度像素
border-radius:10px;        //边框圆角半径像素
border-color:rgba(255,255,255,30);    //边框padding:6px;                //填衬
                }
                //鼠标按下样式
                QPushButton:pressed{
                background-color:rgba(100,255,100,200);
                border-color:rgba(255,255,255,30);
                border-style:inset;
                color:rgba(0,0,0,100);
                }
                //鼠标悬停样式
                QPushButton:hover{
                background-color:rgba(100,255,100,100);
                border-color:rgba(255,255,255,200);
                color:rgba(0,0,0,200);
                }</string>
           </property>
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
